packages/backend/src/routes/attesterDescription.mjs

```diff
@@ -20,6 +20,7 @@ export default ({ app, db, synchronizer }) => {
       passed = false
     }
 
+    // is nonce necessary here?
```

Comment:
@0xWOLAND i don't have access to a nonce with the signing method used. is it nescessary? not sure why you chose to hash nonce and description. i'm creating a hash (of icon, name, description, url) with the same method in the FE, maybe it's better to just pass the hash i already have and use here?

Reply:
imo nonce is better because then each signature is guaranteed to be unique. i think you could generate a new nonce everytime you sign a message using `nonce = Math.floor(Math.random() * 1000...)` and letting `nonce` be mutable.

Reply:
fixed